### PR TITLE
Make a dedicated script fieldtrial_testing_config generation

### PR DESCRIPTION
**Pull request by atuchin-m**
Resolves https://github.com/brave/brave-variations/issues/409

This PR moves the code about field trials generation to a dedicated .py file.
Also it adds support  trials generation for specific date/branch.

b-c PR: https://github.com/brave/brave-core/pull/16016

### CODE REVIEW

File: seed/fieldtrials_testing_config_generator.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
```
Comment (goodov) [inline seed/fieldtrials_testing_config_generator.py:1]: copyright (with https scheme).

File: seed/fieldtrials_testing_config_generator.py [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+      args.append(f'--before={date}')
+    args.append(f'origin/{branch}')
+    output = subprocess.check_output(args)
+    return output.rstrip().decode('utf-8')
+
+
+def _get_seed_data(seed_git_path: str, variations_revision: str):
+  seed_string = subprocess.check_output(
+      ['git', 'show', f'{variations_revision}:{seed_git_path}'])
+  return json.loads(seed_string)
```
Comment (goodov) [inline seed/fieldtrials_testing_config_generator.py:44]: 2 spaces/4 spaces?

**Review by goodov — APPROVED** [review]

File: seed/fieldtrials_testing_config_generator.py (hunk 2 above)
Comment (atuchin-m) [reply seed/fieldtrials_testing_config_generator.py]: Fixed

File: seed/fieldtrials_testing_config_generator.py (hunk 1 above)
Comment (atuchin-m) [reply seed/fieldtrials_testing_config_generator.py:1]: Added

**Pull request merged**